### PR TITLE
style navbar links

### PR DESCRIPTION
**Pull request by mark-a-wilson**

### CODE REVIEW

**Comment by parulmishra** [comment]
SonarQube Analysis: https://sonarqube-ifttgq-tools.pathfinder.gov.bc.ca/dashboard?id=BCDevExchange-PR-40

File: bcdevexchange/wwwroot/css/style.css [hunk 1]
```diff
@@ -153,7 +153,7 @@ h3 {
 
 .main-header-area {
   background: #181818;
-  line-height: 3em;
+  line-height: 2.3em;
```
Comment (thegentlemanphysicist) [inline bcdevexchange/wwwroot/css/style.css:156]: Just confirming that this is supposed to be 2.3em and not 2em

File: bcdevexchange/wwwroot/css/style.css [hunk 2]
```diff
@@ -164,7 +164,17 @@ h3 {
 
 .header-area .main-header-area .main-menu ul li a {
   color: #fff;
+  opacity: 0.5;
+  font-weight: 400;
+  letter-spacing: 0.3px;
```
Comment (thegentlemanphysicist) [inline bcdevexchange/wwwroot/css/style.css:169]: Is 0.3 pixels actually visible?

**Review by thegentlemanphysicist — COMMENTED** [review]
Just curious about some of these numerical choices for formatting.

File: bcdevexchange/wwwroot/css/style.css (hunk 2 above)
Comment (mark-a-wilson) [reply bcdevexchange/wwwroot/css/style.css:169]: Fair question - it actually is visible, just makes the words slightly less crunched.

File: bcdevexchange/wwwroot/css/style.css (hunk 1 above)
Comment (mark-a-wilson) [reply bcdevexchange/wwwroot/css/style.css:156]: Good call on this one. 3em makes the the navbar too tall for my liking, so reducing to 2.3em helps. However, I've found that removing this style altogether has the same effect - so this line is unnecessary. If you'd like to remove it, please go ahead.

**Review by thegentlemanphysicist — APPROVED** [review]
I'm fine with leaving those as they are, just wanted to make sure I understood them.

**Pull request merged**